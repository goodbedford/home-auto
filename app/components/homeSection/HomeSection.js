import React from "react";


const HomeSection = (props) => {

  return (
    <div className="home-auto__section">
      {props.children}
    </div>
  );
};

export default HomeSection;
