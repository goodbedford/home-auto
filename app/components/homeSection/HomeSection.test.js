/* eslint-disable no-undef */

import React from "react";
import {shallow} from "enzyme";
import toJson from "enzyme-to-json";
import HomeSection from "./HomeSection";

describe("<HomeSection />", () => {
  it("HomeSection snapshot", () => {
    const wrapper = shallow(<HomeSection />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
