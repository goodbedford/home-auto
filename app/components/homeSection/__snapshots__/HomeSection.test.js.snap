// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeSection /> HomeSection snapshot 1`] = `
<div
  className="home-auto__section"
/>
`;
